pom.xml

```diff
@@ -52,8 +52,8 @@
     </modules>
 
     <properties>
-        <hazelcast.git.repo>hazelcast</hazelcast.git.repo>
-        <hazelcast.git.branch>master</hazelcast.git.branch>
+        <hazelcast.git.repo>vbekiaris</hazelcast.git.repo>
```

Comment:
It's necessary for the build to pass, otherwise the build will download the code from `master` that still references `EnterpriseMap*` classes and will not compile. I think it's standard process to do this in a PR and switch back to `hazelcast/master` as soon as `master` changes are merged.